examples/djangopy/djangopy/settings.py

```diff
@@ -27,6 +27,16 @@
 if 'SECRET_KEY' in os.environ:
     SECRET_KEY = os.environ['SECRET_KEY']
     DEBUG = False
+else:
+    # To work locally, you should create a .env
+    # and put your env variables there.
```

Comment:
For local development we actually recommend using the [Railway CLI](https://docs.railway.app/develop/cli) and `railway run` to pull in environment variables. Is there any reason this isn't working for you? Or do you just prefer to use a `.env` file?